src/app/commands/cmd_cut.cpp

```diff
@@ -30,12 +37,18 @@ CutCommand::CutCommand()
 
 bool CutCommand::onEnabled(Context* ctx)
 {
-  return App::instance()->inputChain().canCut(ctx);
+  return App::instance()->isGui() ?
+    App::instance()->inputChain().canCut(ctx) : true;
```

Comment:
Is this true? Or should we check that `ctx->clipboard()`? Should `Context` offer a default input chain?